go.mod

```diff
@@ -1048,32 +1066,17 @@ exclude (
 	github.com/knadh/koanf/providers/confmap v0.1.0
 )
 
-replace (
-	// Stick to v0.28.6 even if trivy want v0.29.x, the way we use trivy shouldn't require any k8s.io packages
-	k8s.io/api => k8s.io/api v0.28.6
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.28.6
-	k8s.io/apimachinery => k8s.io/apimachinery v0.28.6
-	k8s.io/apiserver => k8s.io/apiserver v0.28.6
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.28.6
-	k8s.io/client-go => k8s.io/client-go v0.28.6
-	k8s.io/component-base => k8s.io/component-base v0.28.6
-	k8s.io/kms => k8s.io/kms v0.28.6
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20230901164831-6c774f458599
-	k8s.io/kubectl => k8s.io/kubectl v0.28.6
-	k8s.io/metrics => k8s.io/metrics v0.28.6
-)
-
 // Prevent dependencies to be bumped by Trivy
-replace (
-	// github.com/DataDog/aptly@v1.5.3 depends on gopenpgp/v2, so we use latest version of go-crypto before the move to gopenpgp/v3
-	github.com/ProtonMail/go-crypto => github.com/ProtonMail/go-crypto v1.0.0
-
-	// k8s.io/component-base@v0.28.6 depends on github.com/prometheus/common@v0.46.0 and github.com/prometheus/client_golang@1.18.0
-	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.18.0
-	github.com/prometheus/common => github.com/prometheus/common v0.46.0
-)
+// github.com/DataDog/aptly@v1.5.3 depends on gopenpgp/v2, so we use latest version of go-crypto before the move to gopenpgp/v3
+replace github.com/ProtonMail/go-crypto => github.com/ProtonMail/go-crypto v1.0.0
 
 // Prevent a false-positive detection by the Google and Ikarus security vendors on VirusTotal
 exclude go.opentelemetry.io/proto/otlp v1.1.0
 
 replace github.com/google/gopacket v1.1.19 => github.com/DataDog/gopacket v0.0.0-20240626205202-4ac4cee31f14
+
+// Remove once https://github.com/kubernetes-sigs/custom-metrics-apiserver/pull/184 is merged
+replace sigs.k8s.io/custom-metrics-apiserver v1.30.0 => github.com/L3n41c/custom-metrics-apiserver v0.0.0-20241014100211-ccded82d0da2
+
+// Remove once https://github.com/kubernetes/kube-state-metrics/pull/2535 is merged
+replace k8s.io/kube-state-metrics/v2 v2.13.0 => github.com/L3n41c/kube-state-metrics/v2 v2.13.1-0.20241025121156-110f03d7331f
```

Comment:
This is the commit of kubernetes/kube-state-metrics#2535.

To be updated as soon as this PR is merged.